lib/suspenders/app_builder.rb

```diff
@@ -222,7 +222,11 @@ def configure_time_formats
     end
 
     def configure_rack_timeout
-      copy_file 'rack_timeout.rb', 'config/initializers/rack_timeout.rb'
+      rack_timeout_config = <<-RUBY
+Rack::Timeout.timeout = (ENV["RACK_TIMEOUT"] || 10).to_i
+      RUBY
+
+      append_file 'config/environments/production.rb', rack_timeout_config
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.